### ISSUE TITLE
Calibration at  midpoint is not same as at 50% duration for Polymarket 

### ISSUE THREAD

**Issue by uhbif19** [wasabipesto/themis#13]
Distibutions for % market duration seems suspicious to me.

In particular AFAIU 50% and midpont should be same, but they are very different:

![Screenshot_20241208_165057_Chrome](https://github.com/user-attachments/assets/7708207b-c86c-4ae9-9193-01b3da4b2280)

![Screenshot_20241208_165410_Chrome](https://github.com/user-attachments/assets/5f5da787-46ef-4ee8-abdd-dbd68b7fe2a8)

**Comment by wasabipesto** [comment]
You are correct, this is certainly a bug. Thanks for submitting, I'll look into this soon.

**Comment by uhbif19** [comment]
Okay, thanks.
